File: test/fixtures/php-guzzle/get_cookie_with_empty_value.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->get('http://localhost:28139/cookies', [
    'headers' => [
        'accept' => 'application/json',
        'Cookie' => 'mysamplecookie=someValue; emptycookie=; otherCookie=2'
    ]
]);

File: test/fixtures/ruby-httparty/get_cookie_with_empty_value.rb
require 'httparty'

url = 'http://localhost:28139/cookies'
headers = {
  'accept': 'application/json',
  'Cookie': 'mysamplecookie=someValue; emptycookie=; otherCookie=2',
}
res = HTTParty.get(url, headers: headers)
